app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,12 @@
+from abc import ABC
+from app.players.player import Player
```

Comment:
You should put a blank line between different groups of imports according to PEP-8. Do not forget to do it in the future